.github/workflows/unittests.yaml

```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 
 jobs:
   ctests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
@SamuelDegelia-NOAA  Could you explain why the changes of the virtual machine type would resolve this issue?

Reply:
@TingLei-NOAA I am not sure why it fixes it. The error was related to pip, so I am guessing that there was an update to the latest default ubuntu version that broke pip. @CoryMartin-NOAA might have more information.

Reply:
@SamuelDegelia-NOAA 's hunch is the same as mine. I'd rather just deal with this later when that version of the virtual machine is no longer available.

Reply:
@SamuelDegelia-NOAA @CoryMartin-NOAA  Thanks for this explanation.